internal/pubsub/acks_test.go

```diff
@@ -175,17 +175,18 @@ func TestTooManyAcksForASingleBatchGoIntoMultipleBatches(t *testing.T) {
 }
 
 func TestAckDoesNotBlock(t *testing.T) {
-	ctx := context.Background()
+	ctx, cancel := context.WithCancel(context.Background())
 	m := &driver.Message{}
 	ds := &ackingDriverSub{
 		q: []*driver.Message{m},
 		sendAcks: func(_ context.Context, ackIDs []driver.AckID) error {
-			select {}
+			<-ctx.Done()
 			return nil
 		},
 	}
 	sub := pubsub.NewSubscription(ds)
 	defer sub.Close()
```

Comment:
Could you add a comment here explaining how the test works? It's not obvious.

IIUC, `sendAcks` will block forever if `cancel` isn't called, so the test is verifying that it can call `Ack` and not block on that call.

Reply:
Yes, that's how it works. Added a comment.